Common/src/CConfig.cpp

```diff
@@ -2022,23 +1994,22 @@ void CConfig::SetConfig_Options() {
   addStringListOption("MARKER_MOVING", nMarker_Moving, Marker_Moving);
   /* DESCRIPTION: Mach number (non-dimensional, based on the mesh velocity and freestream vals.) */
   addDoubleOption("MACH_MOTION", Mach_Motion, 0.0);
-  default_vel_inf[0] = 0.0; default_vel_inf[1] = 0.0; default_vel_inf[2] = 0.0;
   /* DESCRIPTION: Coordinates of the rigid motion origin */
-  addDoubleArrayOption("MOTION_ORIGIN", 3, Motion_Origin, default_vel_inf);
+  addDoubleArrayOption("MOTION_ORIGIN", 3, Motion_Origin);
```

Comment:
@TobiKattmann the default velocity was not 1,0,0 because it was being re-used here as the default value for something else.

This trips a lot of people (myself included) but the actual parsing of the options does not take place in this method, it creates a list of option parsing objects but it does not read immediately. Hence the need for the config postprocessing method to check for conflicting things and so on.